src/components/AddDependencySearchResult/AddDependencySearchResult.js

```diff
@@ -127,20 +128,54 @@ class AddDependencySearchResult extends PureComponent<Props> {
 
     const downloadNumColor = getColorForDownloadNumber(hit.downloadsLast30Days);
 
+    // We have to use a custom highlight
+    // to unescape the html
+    const CustomHighlight = connectHighlight(
```

Comment:
I think we should move this to its own file, and make it clearer that it's a component.

I'd create a `CustomHighlight` component in `src/components`, and restructure it like:

```js
type Props = any; // If we really wanted to we could add types for this, but it's 3rd-party so it doesn't feel critical to me. But let's still create a Props type, for consistency.

const CustomHighlight = ({ highlight, attribute, hit, highlightProperty }: Props) => {
  ... All the stuff you have here
}

export default connectHighlight(CustomHighlight);
```

This project has a fair amount of "boilerplate" for new components (their own directory, an `index.js` that forwards exports...), but you can create them automatically with [new-component](https://github.com/joshwcomeau/new-component)